src/pages/ImportData.js

```diff
@@ -368,6 +380,16 @@ export const isProtected = (url, filetype) => {
   }
 };
 
+// This method identifies whether a workspace qualifies as protected.
+// 'Protected' here means that it has enhanced logging - either on its own or because it has an auth domain.
+// For now this also means only GCP workspaces are included.
+export const isProtectedWorkspace = (workspace) => {
+  if (!isGoogleWorkspace(workspace)) {
+    return false;
+  }
+  return !!workspace.workspace.bucketName && workspace.workspace.bucketName.startsWith('fc-secure');
```

Comment:
I thought at this point you already had `workspace.workspace` (from `WorkspaceSelector`). So I am confused about `workspace.workspace.bucketName`.

Reply:
`workspace.workspace` has been confusing me for basically my entire life in terra-ui.  But `isGoogleWorkspace` expects its input to be a workspace with an inner `workspace` object, so I've passed along that workspace object from the `WorkspaceSelector`

Reply:
But isn't `workspace` coming from https://github.com/DataBiosphere/terra-ui/pull/4019/files#diff-6ad2c2eca254b516f619176e9b3d0d7449612d0f612d13e99f629ac726a2b262R103, in which case it would just be `workspace.bucketName`?

Reply:
Note also the existence of `isGoogleWorkspaceInfo`.

Reply:
Got it now, thanks!